src/cli/commands/public-cmds/login-cmd.js

```diff
@@ -9,14 +9,16 @@ export default class Login extends Command {
   alias = '';
   opts = [
     ['p', 'port <port>', 'port number to open for localhost server (default 8085)'],
-    ['', 'no-launch-browser', 'do not open a browser for authentication']
+    ['', 'no-launch-browser', 'do not open a browser for authentication'],
+    ['r', 'rc-path <path>', 'path to npmrc file to configure bitsrc registry'],
+    ['', 'skip-registry-config [boolean]', 'dont configure bitsrc registry'],
```

Comment:
keep in mind that if you put the [boolean] here, i think the user must enter true or false

Reply:
i check this and the user only needs to write --skip-registry-config and the value is true